ui/packages/catalog-ui-search/src/main/webapp/react-component/location/keyword.js

```diff
@@ -16,6 +16,21 @@ class Keyword extends React.Component {
     }
     this.fetch = this.props.fetch || fetch
   }
+  extractGeo(suggestion) {
+    return {
+      type: 'Feature',
+      geometry: {
+        type: 'Polygon',
+        coordinates: [
+          suggestion.geo.map(coord => {
```

Comment:
I'm not 100% certain, but I think you can simplify this a bit to `suggestion.geo.map(coord => [coord.lat, coord.lon])`

Reply:
You *might* have to put parens around the square brackets, but I thought was only needed when returning an object.

Reply:
Do you need the outer brackets for ```coordinates```?

Reply:
Good call on the simplification. 👍 

Regarding the outer brackets, yes, we need them. The expected data structure by the handler of the `geofeature(id)` function involves 3 layers of arrays. Here's an example: 

_http://localhost:8080/search/catalog/internal/geofeature?id=3210000365e84e8cb8fe07698b6446e7_

```
{
    "type": "Feature",
    "geometry": {
        "type": "Polygon",
        "coordinates": [
            [
                [
                    48.1166,
                    -13.5472
                ],
                [
                    48.4166,
                    -13.5472
                ],
                [
                    48.4166,
                    -13.2472
                ],
                [
                    48.1166,
                    -13.2472
                ],
                [
                    48.1166,
                    -13.5472
                ]
            ]
        ]
    },
    "properties": {},
    "id": "Hell-Ville, MDG"
}
```